docs/rules/no-restricted-modules.md

```diff
@@ -32,23 +38,37 @@ You may also specify a custom message for any paths you want to restrict as foll
 "no-restricted-modules": ["error", [{
   "name": "foo-module",
   "message": "Please use bar-module instead."
-}]]
+  }]
+]
 ```
 
 or like this:
 
 ```json
-"no-restricted-modules": ["error", {
-  "paths": [{
-    "name": "foo-module",
-    "message": "Please use bar-module instead."
+"no-restricted-modules": ["error",{
+"paths":[{
+  "name": "foo-module",
+  "message": "Please use bar-module instead."
   }]
 }]
 ```
 
 The custom message will be appended to the default error message. Please note that you may not specify custom error messages for restricted patterns as a particular module may match more than one pattern.
 
-Examples of **incorrect** code for this rule with sample `"fs", "cluster"` restricted modules:
+
+To restrict the use of all Node.js core modules (via https://github.com/nodejs/node/tree/master/lib):
+
+```json
+{
+    "no-restricted-modules": ["error",
+        "assert","buffer","child_process","cluster","crypto","dgram","dns","domain","events","freelist","fs","http","https","module","net","os","path","punycode","querystring","readline","repl","smalloc","stream","string_decoder","sys","timers","tls","tracing","tty","url","util","vm","zlib"
+    ]
+}
+```
+
+## Examples
+
+Examples of **incorrect** code for this rule  with sample `"fs", "cluster"` restricted modules:
```

Comment:
Nitpick: should also add "lodash/*"? the 3rd example used.

Reply:
+ https://github.com/eslint/eslint/pull/9495/files#diff-47f659a4e87393de269d9c0857e7da7eR89
+ https://github.com/eslint/eslint/pull/9495/files#diff-47f659a4e87393de269d9c0857e7da7eR107

also, I found the variable names in these lines are a little misleading, could you help to improve? ❤️

Reply:
these names were the ones used in the original doc also,if you want i can fix it up in no time

Reply:
as the title "Fix:improve the doc for no-restricted-rule", I guess it is fine to update in this PR. 😄  thanks!

Reply:
done with that,if any other problem do tell